cirq/ion/ion_gates_test.py

```diff
@@ -44,9 +48,12 @@ def test_ms_matrix():
 
 
 def test_ms_repr():
+    ms = cirq.ms(np.pi / 2)
```

Comment:
nit: moved down to right where this is used to make it more readable.  I was wondering what you were doing with it for a while ;)

Reply:
done